src/dataframe/sort.jl

```diff
@@ -4,13 +4,14 @@
           alg::Union{Algorithm, Nothing}=nothing, lt=isless, by=identity,
           rev::Bool=false, order::Ordering=Forward)
 
-Sort data frame `df` by column(s) `cols`.
+Sort data frame `df` by column(s) `cols`. The sorting on multiple columns is done lexicographicallly.
```

Comment:
is "The" needed before "sorting"?

Also you might add the same comment in `sort` above.

Reply:
Yeah, that isn't needed. I'll remove this and will add the comment to the `sort`.